storage/series/types.go

```diff
@@ -251,4 +253,29 @@ type Options interface {
 
 	// IdentifierPool returns the identifierPool
 	IdentifierPool() ident.Pool
+
+	// SetStats sets the configured Stats.
+	SetStats(value Stats) Options
+
+	// Stats returns the configured Stats.
+	Stats() Stats
+}
+
+// Stats is passed down from above to avoid constructing tally constructs per series/buffer.
```

Comment:
"passed down from above" sounds like scripture haha, can you change this to "passed down from the namespace/shard objects to avoid an allocation per series" or something like that

Reply:
Hah amen.